lib/daru.rb

```diff
@@ -26,19 +26,20 @@ module Daru
     10 => 31,
     11 => 30,
     12 => 31
-  }
+  }.freeze
 
-  SPLIT_TOKEN = ','
-  class << self
-    @@lazy_update = false
+  @lazy_update = false
+
+  SPLIT_TOKEN = ','.freeze
```

Comment:
If it can be changed by user, it should not be a constant, obviously.  Though, after `.freeze`, your example still working. What is not working, is things like `Daru::SPLIT_TOKEN << 'xxx'`